ci-operator/config/maistra/test-infra/maistra-test-infra-main.yaml

```diff
@@ -0,0 +1,68 @@
+base_images:
+  cli:
+    name: "4.10"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: maistra-builder
+    namespace: ci
+    tag: "2.4"
+  use_build_cache: true
+releases:
+  latest:
+    release:
+      channel: stable
+      version: "4.10"
+resources:
+  '*':
+    limits:
+      memory: 16Gi
+    requests:
+      cpu: "2"
+      memory: 4Gi
+tests:
+- as: sync-upstream-envoy-1.22
+  commands: |
+    ./tools/automator-main.sh \
+    --org=maistra \
+    --repo=istio \
+    --branch=maistra-2.3 \
+    --token-path=/creds-github/token \
+    '--title=Automator: merge upstream changes to $AUTOMATOR_ORG/$AUTOMATOR_REPO@$AUTOMATOR_BRANCH' \
+    "--labels=merge strategy" \
+    --modifier=merge_upstream_istio_master \
+    --merge-repository=https://github.com/envoyproxy/envoy.git \
+    --merge-branch=master \
+    --merge-failure-notify=@maistra/release-managers-2-3 \
+    --merge-strategy=merge
+  container:
+    from: src
+  cron: 00 05 * * 1-5
+  secrets:
+  - mount_path: /creds-github
+    name: github-token
+- as: sync-upstream-envoy-1.24
```

Comment:
Instead of using the upstream branch, I'd use ours, and would call this job `sync-upstream-envoy-2.4`

Reply:
done